lib/active_merchant/billing/gateways/cyber_source_rest.rb

```diff
@@ -93,6 +98,7 @@ def scrub(transcript)
           gsub(/(\\?"number\\?":\\?")\d+/, '\1[FILTERED]').
           gsub(/(\\?"routingNumber\\?":\\?")\d+/, '\1[FILTERED]').
           gsub(/(\\?"securityCode\\?":\\?")\d+/, '\1[FILTERED]').
+          gsub(/(\\?"cryptogram\\?":\\?")[^<]+/, '\1[FILTERED]').
```

Comment:
@almalee24 also added scrubbing of cryptogram and a test for it.

Reply:
Got it, thanks for catching that!